src/AltinnCore/UnitTest/Designer/Services/ReleaseServiceTests.cs

```diff
@@ -0,0 +1,97 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AltinnCore.Common.Services.Interfaces;
+using AltinnCore.Designer.Infrastructure.Models;
+using AltinnCore.Designer.Repository;
+using AltinnCore.Designer.Repository.Models;
+using AltinnCore.Designer.Services;
+using AltinnCore.Designer.TypedHttpClients.AzureDevOps;
+using AltinnCore.Designer.TypedHttpClients.AzureDevOps.Enums;
+using AltinnCore.Designer.TypedHttpClients.AzureDevOps.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.Documents;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+
+namespace AltinnCore.UnitTest.Designer.Services
+{
+    /// <summary>
+    /// Tests for ReleaseService
+    /// </summary>
+    public class ReleaseServiceTests
+    {
+        private readonly ReleaseService _sut;
+        private readonly Mock<ReleaseRepository> _releaseDbRepoMock;
+        private readonly Mock<IAzureDevOpsBuildService> _azureDevOpsBuildServiceMock;
+        private readonly Mock<ISourceControl> _sourceControlMock;
+        private readonly Mock<IGitea> _giteaWrapperMock;
+
+        /// <summary>
+        /// x
+        /// </summary>
+        public ReleaseServiceTests()
+        {
+            _releaseDbRepoMock = new Mock<ReleaseRepository>();
+            _azureDevOpsBuildServiceMock = new Mock<IAzureDevOpsBuildService>();
+            _sourceControlMock = new Mock<ISourceControl>();
+            _giteaWrapperMock = new Mock<IGitea>();
+
+            Mock<IHttpContextAccessor> httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            DefaultHttpContext context = new DefaultHttpContext();
+            httpContextAccessorMock.Setup(_ => _.HttpContext).Returns(context);
+            AzureDevOpsSettings azureDevOpsSettings = new AzureDevOpsSettings();
+            Mock<OptionsMonitor<AzureDevOpsSettings>> optionsMonitorMock = new Mock<OptionsMonitor<AzureDevOpsSettings>>();
+            optionsMonitorMock.Setup(x => x.CurrentValue).Returns(azureDevOpsSettings);
+
+            _sut = new ReleaseService(
+                _releaseDbRepoMock.Object,
+                httpContextAccessorMock.Object,
+                _azureDevOpsBuildServiceMock.Object,
+                _sourceControlMock.Object,
+                optionsMonitorMock.Object);
+        }
+
+        /// <summary>
+        /// x
```

Comment:
x

Reply:
Is stylecop enabled on the unittest project ? We should consider making the rules less strict så that we don't need as much documentation for the test classes.

Reply:
X gun give it to ya. Fixing :)

Reply:
I agree with you regarding less strictness